app/app.py

```diff
@@ -1,7 +1,9 @@
 from flask import Flask
 from flask_migrate import Migrate
+from app.models.mega_sena import MegaSena
```

Comment:
O import da classe `MegaSena` pode ser simplificado.

```suggestion
from app.models import MegaSena
```